media/js/firefox/whatsnew/whatsnew-112-eu-privacy.js

```diff
@@ -0,0 +1,85 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+function defaultTrue() {
+    'use strict';
+
+    document.querySelector('.wnp-cta-default').classList.add('hide');
+    document.querySelector('.wnp-cta-protections').classList.add('show');
+
+    // Open about:protections using UITour when button is clicked.
+    document
+        .querySelector('.wnp-cta-protections > .mzp-c-button')
+        .addEventListener(
+            'click',
+            (e) => {
```

Comment:
I had no idea that we could use arrow functions outside of the .es6.js files, TIL

Reply:
Ahhhh, still have morning brain fog